internal/jobs/twitter/auth.go

```diff
@@ -36,6 +37,8 @@ func NewScraper(config AuthConfig) *Scraper {
 			logrus.Debugf("Already logged in as %s.", config.Account.Username)
 			return scraper
 		}
+	} else {
+		logrus.Warnf("Failed to load cookies for user %s: %v", config.Account.Username, err)
```

Comment:
💯

Reply:
yeah... was testing and that was silently failing b/c the volume wasn't mounted -> painful